packages/server/graphql/mutations/helpers/pushEstimateToGitLab.ts

```diff
@@ -0,0 +1,77 @@
+import {GraphQLResolveInfo} from 'graphql'
+import {SprintPokerDefaults} from 'parabol-client/types/constEnums'
+import makeAppURL from 'parabol-client/utils/makeAppURL'
+import appOrigin from '../../../appOrigin'
+import GitLabServerManager from '../../../integrations/gitlab/GitLabServerManager'
+import {GetIssueQuery} from '../../../types/gitlabTypes'
+import getPhase from '../../../utils/getPhase'
+import makeScoreGitLabComment from '../../../utils/makeScoreGitLabComment'
+import {GQLContext} from '../../graphql'
+import createNote from '../../nestedSchema/GitLab/mutations/createNote.graphql'
+import getIssue from '../../nestedSchema/GitLab/queries/getIssue.graphql'
+import {ITaskEstimateInput} from '../../types/TaskEstimateInput'
+
+const pushEstimateToGitLab = async (
+  taskEstimate: ITaskEstimateInput,
+  context: GQLContext,
+  info: GraphQLResolveInfo,
+  stageId: string
+) => {
+  const {dimensionName, taskId, value, meetingId} = taskEstimate
+  const {dataLoader} = context
+  const [task, meeting] = await Promise.all([
+    dataLoader.get('tasks').load(taskId),
+    dataLoader.get('newMeetings').load(meetingId)
+  ])
+  if (!meeting) return new Error('Meeting does not exist')
+  const gitlabIntegration = task.integration as Extract<
+    typeof task.integration,
+    {service: 'gitlab'}
+  >
+  const {teamId} = task
+  const {accessUserId, gid} = gitlabIntegration
+  const auth = await dataLoader.get('teamMemberIntegrationAuths').load({
+    service: 'gitlab',
+    teamId,
+    userId: accessUserId
+  })
+  if (!auth) return new Error('User no longer has access to GitLab')
+  const {accessToken, providerId} = auth
+  const provider = await dataLoader.get('integrationProviders').load(providerId)
+  if (!provider) return new Error('Integration provider not found')
+  const manager = new GitLabServerManager(accessToken!, provider.serverBaseUrl!)
+  const gitlabRequest = manager.getGitLabRequest(info, context)
+
+  const [data] = await gitlabRequest<GetIssueQuery>(getIssue, {gid})
```

Comment:
In this PR https://github.com/ParabolInc/parabol/pull/6427 this is refactored to use the `TaskIntegrationManager` pattern